src/test/java/ServiceObjects/HomePage.java

```diff
@@ -46,6 +49,32 @@ public void titleOfPage(String title) {
     public void checkSearch(String searchTerm) {
         response = httpRequest.queryParam("controller", "search").queryParam("s", searchTerm).when().get();
         response.then().assertThat().statusCode(200);
-        Assertions.assertEquals("Search results",xmlParseByAttribute("class","h2"));
+        Assertions.assertEquals("Search results", xmlParseByAttribute("class", "h2"));
+    }
+
+    public void login(String userName, String password) {
+        getCookiesFilter();
+        Response response = httpRequest
+                .contentType("multipart/form-data")
+                .queryParam("controller", "authentication")
+                .queryParam("back", "my-account")
+                .accept("text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9")
+                .redirects().follow(true)
+                .multiPart("token", "8b3ee4f1cba54a1031be9bb7879e9dad")
+                .multiPart("back", "my-account")
+                .multiPart("email", "test@123.com")
+                .multiPart("password", "test123")
```

Comment:
"test123.com" and "test123" should be replaced by userName and password so the method parameters are used